### DISCUSSION TITLE
evince: cannot access evolution attachments in /tmp

### DISCUSSION THREAD

**Discussion by githlp** [netblue30/firejail#5101] [Q&A] [answered]
the evolution app seems to provide attachements not only in
`${HOME}/.cache/evolution/`
but also in
`/tmp/evolution-ronald-ROHBK1/`

How do i allow the /tmp dir in evince.local ?

I guess the last part of path `evolution-ronald-ROHBK1/` will change with different attachements.. 

Relates to:

* #6469

**Comment by rusty-snake** [comment, marked as answer]
`whitelist /tmp/evolution-*` or `ignore private-tmp`
